pkg/client/events/replay_client.go

```diff
@@ -21,7 +21,7 @@ const (
 	// eventsBytesRetryLimit is the maximum number of times to attempt to
 	// re-establish the events query bytes subscription when the events bytes
 	// observable returns an error or closes.
-	eventsBytesRetryLimit        = 10
+	eventsBytesRetryLimit        = 0
```

Comment:
Can we add a TODO to make this a customisable parameter in the appgateserver and relayminer config files. It would reqiire some refactoring to make it variable not a fixed value but could provide some benefits.